requirements/extra.txt

```diff
@@ -7,4 +7,5 @@ torchtext>=0.5
 onnx>=1.7.0
 onnxruntime>=1.3.0
 hydra-core>=1.0
+git+https://github.com/OpenMined/PySyft.git@refs/pull/5221/head
```

Comment:
is there a plan to merge that PR before this one?

Reply:
Possibly.

Reply:
i wouldn't merge this until the PR is merged.

Or at least, i'd ask their team to create a branch in their repository with these changes. This would also allow installing a `.zip` as @Borda requested